Makefile

```diff
@@ -58,12 +58,11 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-test: manifests generate fmt vet ## Run unit tests
-		go test \
-		-race \
-		-covermode=atomic \
-		-coverprofile coverage.out \
-		./...
+ENVTEST_ASSETS_DIR ?= $(shell pwd)/testbin
+test: manifests generate fmt vet ## Run tests.
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
```

Comment:
I noticed k8s_version default is set 1.19.2 - should we set this variable (i.e to later version 1.21.1 used in version 4.8.12 of openshift) ?

Reply:
Where do you see the version being hardcoded? Do you mean the `controller-runtime` version used here?

Reply:
in the script setup-envtest.sh  -> k8s_version="${ENVTEST_K8S_VERSION:-1.19.2}" - that's the default. I was asking if we need to set the envar ENVTEST_K8S_VERSION

Reply:
Looks like there is a [tool](https://github.com/kubernetes-sigs/controller-runtime/tree/master/tools/setup-envtest) for managing the binaries for the envtests. I've switched the make target and set the version.